src/integration/blockchain/ain/node/defi-client.ts

```diff
@@ -2,6 +2,7 @@ import { AccountHistory, AccountResult, UTXO as SpendUTXO } from '@defichain/jel
 import { SchedulerRegistry } from '@nestjs/schedule';
 import BigNumber from 'bignumber.js';
 import { HttpService } from 'src/shared/services/http.service';
+import { Proposal, ProposalVote } from 'src/subdomains/core/statistic/cfp.service';
```

Comment:
I think `Proposal` and `ProposalVote` need to be placed in blockchain folder, not in CFP.